### ISSUE TITLE
I need to rename this project…

### ISSUE THREAD

**Issue by csswizardry** [inuitcss/inuitcss#426]
Whichever way you slice it, the name _inuitcss_, and especially the logo, is patronising at best. The implementation reduces an entire people to a literal caricature—this is entirely and only my fault.

I’ve been asked many times over the last almost-decade where the name came from and the honest answer is: I have no idea. There’s no wordplay, no association, nothing faux-witty or amusing. Nothing at all. I’m happy to see it go. In fact, thinking about the name nowadays embarrasses me.

Labelled with _enhancement_ as dropping this name will only make the project better, but I’m happy for more input and triage here. Also, I’ve assigned the entire core team to this issue for their input and steer (honestly, I have nothing to do with the day-to-day of this project anymore), but it bears reiterating that the choice of name and the commissioning of the logo was entirely my doing.

* Question: What do the core team think about renaming?

**Alternatives**

Thankfully, I think the alternative is already set up for us: ITCSS. I’ve always described ITCSS as the methodology and inuitcss as an OSS manifestation of it, so they’re already intrinsically linked with one another. Renaming inuitcss to ITCSS merely formalises that a little more, and has the added benefit of being an already-established moniker. We don’t have to start from zero.

* Question: What do the core team think about renaming to ITCSS?

**Logistics**

I’m not overly concerned—more popular projects have undertaken larger rebrands—but it would be wise to have a plan. Domains and GitHub accounts for ITCSS are already secured—perhaps someone can leap on npm if we want to move over to the ITCSS brand?

* Question: What do we need to consider when we go ahead with this?

**Branding**

I would love to get an OSS designer involved in developing a logo and brand for ITCSS. I partially commissioned a designer for ITCSS many years ago, but it was never really complete.

* Question: Would any designers like to contribute to this change?

**N.B.** It’s my birthday today so I’m not going to actively participate in much today or the next few days—I just wanted to get the thoughts out of my head and in front of the community.

Much love!  
H

**Comment by yefim** [comment]
UPDATE: I am removing my suggestion and I am sorry for putting it up in the first place. I learned about the symbol from the news last month but did not connect the dots to my suggestion. Thank you @ndorfin for pointing it out. I'm including my actions that led to my suggestion.

<details>
 <summary>What I did</summary>
  <p>
    <ul>
      <li>I googled what ITCSS stood for.</li>
     <li>I found out it was Inverted Triangle CSS and saw a picture of an inverted triangle on <a href="https://www.creativebloq.com/web-design/manage-large-css-projects-itcss-101517528">this blog post</a> (SFW).</li>
    <li>I thought an animated inverted triangle would be a good logo.</li>
    <li>I googled "cartoon upside down triangle" and posted the first picture I found.</li>
    </ul>
</p>
</details>

**Comment by Saurou** [comment]
Unify names and just use one ( ITCSS ) it's a good idea, and less confusing, although I think that inuitcss it's easier to remember as you associate a figure and more memorable name to it.
In any case, I don't think you should feel guilty or ashamed for the very unoffensive (actually really cute) image and name you've been using.
I hope no one has been annoying you, especially if just to abuse these particular times we're living.

**Comment by csswizardry** [comment]
> …although I think that inuitcss it's easier to remember as you associate a figure and more memorable name to it.

This is certainly true—it’s proved a very strong brand! But it’s not the only way to achieve it, and certainly shouldn’t be at the expense of appropriation.

> …I don't think you should feel guilty or ashamed for the very unoffensive…

Thanks for your support. The thing is, I _do_ feel embarrassed and guilty. I haven’t been strong-armed or forced into this situation—it really doesn’t sit well with me.

> …(actually really cute)…

This is exactly my issue: it’s patronising at best. Cutesifying indigenous people for a CSS framework isn’t a good move!

> I hope no one has been annoying you…

Remarkably, no. Though I expected and would have deserved it.

As an aside, sorry to see you’re getting downvotes. If I understand your sentiment correctly, you'd actually support a name-change while also looking out for me. Your tone to me seems reasonable and pleasant. Thank you.

**Comment by sassquad** [comment]
I have some open thoughts.

- Rather than inuit, how about intuitive, or 'intuit' CSS? You would still need a logo, but how would you visualise such a word?
- play on the triangle methodology a bit more. What can an inverted triangle resemble? Alternative names for a pyramid, perhaps?

As long as links and URLs redirect to the new named framework, you reduce the risk of losing an established audience, or mistakenly believe the framework has stopped being supported.

Enjoy your birthday, and look forward to seeing what the new project name will be!

**Comment by Saurou** [comment]
Ohh thanks, they're just downvotes so no problem :)

It's nice to hear these are your genuine thoughts, the most important thing.

A search for a new name / logo will also be exciting and trigger people imagination.

And Happy Birthday!

**Comment by ndorfin** [comment]
> What about an inverted triangle cartoon character for a logo? Something like
> ![image](https://user-images.githubusercontent.com/1170843/86510169-3e8cfc00-bda2-11ea-8afc-f2c62d3dcb49.png)
> 
> source: https://drawception.com/panel/drawing/5ezt3336/a-happy-red-triangle-man/

Eeek! Avoid.
See: 

- https://www.fastcompany.com/90518919/yes-the-trump-campaign-used-a-nazi-symbol-in-its-ads-heres-what-it-means
- https://www.vox.com/recode/2020/6/18/21295226/facebook-trump-campaign-nazi-symbol-antifa

**Comment by fuhlig** [comment]
The intention of changing the name is understandable.
While not having a suggestion for an alternative / new name, I wouldn't go with ITCSS. 

ITCSS is an established css architecture. While inuitcss uses the architecture, it is also tied a lot to the implementation details and tooling, e.g. 
- Sass
- structure & naming & inheritance of (Sass) variables & module settings 
- naming conventions: module naming, naming convention (BEM)

Just to name a few, but none is part of the core behind ITCSS. 

An example of naming architecture / methodology and tools / implementations the same is gitflow. In my experience this led to a lot of confusion for development teams. 
Gitflow is a **methodology** for a git workflow but also a **tool** for predefined/customizable for "enforcing" a workflow.

> you're using gitflow?
> yep!
> Where can I see the configuration? Can't get it to run...
> OH... You mean the tool? Yeah no we are using gitflow as a convention & mental model while being agnosting on how to use it.

> you're using gitflow?
> no
> Okay... What is your branching strategy?
> feature, bugfix, hotfix.......
> So basically gitflow?
> If you want to call it that way. But we're doing it via this plugin.
> 😐

**Comment by LayZeeDK** [comment]
Now that we're in the conversation about offending groups of people, consider revisiting "Inverted Triangle".

>Inverted triangles were used by the Nazis to classify prisoners during the Second World War. The upside down triangles were used as identifying badges, and were sewn into the uniforms of concentration camp prisoners.

**Comment by anselmh** [comment]
I’m very fine with ITCSS and it is probably the best idea to use this naming for the framework. It’s the implementation of the underlying methodology so I think it’s fair to provide the implementation under the same naming. I don’t think this does require a specific technology stack, vice-versa it opens up the idea of providing a scss, a postcss, a less and other versions of the framework in future. But also I have to say I’m not actively contributing much these days as well anymore. Let’s see what the rest of our team says.

**Comment by herzinger** [comment]
@anselmh no one is, the project is virtually frozen for a while.

About the topic at hand, I agree with the significance of a name / branding change, and while I also agree while @fuhlig about the confusion of having the methodology and tooling with the same name, I'm not totally opposed to using ITCSS. Also, if that is decided to be the case, I don't think that worrying about how the nazis used inverted triangles should matter in this context, seeing as 1. it's a basic shape, you can find thousands of ways it has been used in history; and 2. it's literally the name of the methodology.

When the decision is made, I can offer some design proposals, as I'm both a designer and deeply familiar with the project, being a contributor from pretty much the start. It would make me even happier if the change included some code review and update and a move toward having proper documentation outside the code base.

**Comment by csshugs** [comment]
Renaming? Yes please!

Renaming to ITCSS? No. As @fuhlig already said, the methodology and the framework is **not the same thing** and thus shouldn’t carry the same name. GitFlow is a good example of introducing much confusion with this.

However, I guess it would make sense to keep the new name close to the (inverted) triangle idea. Something like “pyramidCSS” would be very well suited IMO.

**Comment by anselmh** [comment]
There’s simply no need to choose a logo that’s similar to a currently (by Mr. T.) and in the past (by Nazis) misused shape. Why would we? There are probably a lot of ways to avoid this context in the logo. It’s relatively easy by, for example not choosing solely a triangle as logo but adding something to it, by not choosing red as the color, etc.

> "pyramidCSS"

A pyramid, for example, could make a nice logo.

**Comment by LayZeeDK** [comment]
>I don't think that worrying about how the nazis used inverted triangles should matter in this context, seeing as 1. it's a basic shape, you can find thousands of ways it has been used in history; and 2. it's literally the name of the methodology.

1. That goes for [swastikas as well](https://en.wikipedia.org/wiki/Western_use_of_the_swastika_in_the_early_20th_century).
2. inuitcss is also currently the name of this library. That doesn't change the discussion at hand.

If we want to go through this trouble, shouldn't we at least try to avoid making the same mistake twice?

**Comment by herzinger** [comment]
> 1. That goes for [swastikas as well](https://en.wikipedia.org/wiki/Western_use_of_the_swastika_in_the_early_20th_century).
> 2. inuitcss is also currently the name of this library. That doesn't change the discussion at hand.

I'm not opposed to using another name, I even agree that it would be better to avoid confusion. Still, a swastika is not a basic shape (in the same sense as triangles, rectangles, circles and etc are), and it has a pretty strong connotation in it's modern iconography that a simple inverted triangle simply doesn't have, specially a subdivided one, like the one used to represent the methodology virtually everywhere in the internet right now. As I said, I will even vote to use a different name. ITCSS was chosen because it describes the logical "shape" of the organizational idea, right?

That being said, I do like having a different name for the framework, but are you suggesting changing the methodology's name as well? Because if so, that should come first. If not, I like inuitcss being renamed to a 3D version of the method's shape, symbolizing how it's a real life application of it, but I would suggest a ziggurat, a stepped, more segmented version of a pyramid, as it literally has "levels".

**Comment by LayZeeDK** [comment]
Yes, I'm suggesting that the name of the methodology could be changed as well.

I find the pyramid idea a little funny as it's the opposite of the methodology's representation, but hey why not? As long as we don't have a cute pyramid-building slave as our mascot.

**Comment by LayZeeDK** [comment]
Just a suggestion: `innocss`/`innoitcss`.

**Comment by herzinger** [comment]
@LayZeeDK yes, I did think about how it's not inverted with a pyramid/ziggurat, but that only depends on how you think about it. I was always under the impression that the same logic for the inverted triangle thing would apply just as much as a normal triangle. What would `inno` stand for?

**Comment by herzinger** [comment]
This a really dirty and fast sketch, but we could use something along this lines, and let the branding visuals conciliate the question of the direction.
![ziggurat concept](https://user-images.githubusercontent.com/1102234/86641808-cbf26b00-bfb1-11ea-9f18-974d2891a0c1.png)

**Comment by LayZeeDK** [comment]
Too detailed to be used in all contexts (small scale), but interesting proposal. Inno = innovation/innovative. innoitcss almost sounds like inuitcss.

**Comment by apsuhos** [comment]
I side with the idea of not mixing the methodology with the framework.

I played with an idea I had in mind when Harry announced his thoughts. I love the circle as a shape because for me it represents completion and flexibility. Inuitcss is a framework that is very mature and pretty extensible if you know what you are doing. That is the reason why I placed the inverted triangle on top of a circle. The framework, the foundation of the methodology, the implementation and the tooling are the circle which support the idea, the architecture. Conveniently, when you subtract a triangle from a circle a "U" shape is created. That is why I chose the name UITCSS. 

I will always love the name - inuitcss - even though I understand Harry's stand. That is why I didn't want it to perish.  uit and inuit. The new name should hide some of the old one's glory inside of it :D

My rough take on this idea.

![uitcss](https://user-images.githubusercontent.com/2563197/87058905-54227b80-c211-11ea-83ff-c4d9d82b1c2f.png)

**Comment by LayZeeDK** [comment]
What about nuitcss? "nu" as in "new"?

**Comment by jozzs** [comment]
I also agree on keeping the methodology as ITCSS and choosing another for the tool. How about "SASSIT"? (Sass Inverted Triangle). Or maybe just keep the initials: SIT

**Comment by herzinger** [comment]
> Too detailed to be used in all contexts (small scale), but interesting proposal. Inno = innovation/innovative. innoitcss almost sounds like inuitcss.
- @LayZeeDK 

As I said, it was just a quick sketch for the concept, not a logo. Scalability and readability are part of the larger design process, which takes time and effort, so I wouldn't do it before we at least define a direction (but it's doable within the idea). 

Some different concepts where put forward, so at this point we need more input from both the core team and @csswizardry himself, I think.

**Comment by hayatbiralem** [comment]
How about getitcss? "Get it CSS" and "Get ITCSS" :D

**Comment by mtsweir** [comment]
@hayatbiralem visually looks like: get tit css

Which makes me think you'd need to then inform people of how to properly pronounce it.

**Comment by mtsweir** [comment]
> I love the circle as a shape because for me it represents completion and flexibility. Inuitcss is a framework that is very mature and pretty extensible if you know what you are doing. That is the reason why I placed the inverted triangle on top of a circle.

@apsuhos should be a little mindful of what the symbol could represent in some people's minds, both good and bad. This logo I thought of as someone's open legs, then had another thought based off your thinking as a triangle cutting through a circle deflating it. Just my feedback though.

**Comment by mtsweir** [comment]
@herzinger have you given some thought to your concepts application? e.g if this symbol was to scale down quite a bit, say a browser icon, how would it look and will it still hold up?

**Comment by mtsweir** [comment]
In terms of design, maybe look at logo designers who do really nice work and get some ideas/inspiration from them. One great designer I follow is [James Martin](https://www.instagram.com/made.by.james/), his work is fantastic and really gets to the core and vision of what clients are after.

BTW, one of his most recent clients he designed a logo in the form of a triangle, he shared his concepts in his Insta feed as well so might be something worth looking at for ideas: [early concepts](https://www.instagram.com/p/CCduJVoFV81/), and [alpha concepts](https://www.instagram.com/p/CB3FLuhlF6a/) - but not saying this is what I prefer, if you have time his work is worth a look at least.

**Comment by herzinger** [comment]
> @herzinger have you given some thought to your concepts application? e.g if this symbol was to scale down quite a bit, say a browser icon, how would it look and will it still hold up?
- @mtsweir 

As I said, properly designing a brand is a lot of work, so no serious designer would start on it before having an idea about name and a rough direction. In a normal project I would allocate hours and resources and work closely with the client for that step as well, but this is a community project, with the actual owner quite distant while still having both final decision and a vision for it, so it's a bit different. What I can ensure is that any final product I eventually got for will have scalability and application in mind.

**Comment by DevJMD** [comment]
> What about nuitcss? "nu" as in "new"?

Ha! "I Knew It" CSS! The kind of "eureka!" moment you have when writing complex CSS and things start to come to life. 😄

**Comment by zomars** [comment]
More than a "pyramid" I see it as a "funnel"

**Comment by mtsweir** [comment]
Just throwing this out there, I had posted this emoji symbol in a DM this morning and it sorta got me thinking about this project. What I like about this is that it's a triangle and the elements are in a continuous flow, I could see an extension of this concept where it creates a similar flow effect in a sort of 3D-negative space approach.

Not saying it has to be this but more just adding some ideas for thought. Hope all is well people.

![Screen Shot 2020-08-27 at 10 25 00 AM](https://user-images.githubusercontent.com/4976660/91362959-a6663d00-e84f-11ea-88a9-fd5cb447db11.png)

**Comment by hayatbiralem** [comment]
How about spelling the word triangle backwards? Elgnairt! Sounds like an ancient or unknown language :)

**Comment by hayatbiralem** [comment]
Considering that mountains are also triangles and the ITCSS methodology is a scalable CSS architecture, I believe we can start with the word mountain.

How about this: IMCSS - Inverted Mountain CSS

It also evokes "I am CSS" and is pronounced like it.

I believe and hope this could be a good brand in many ways.

**Comment by glennmcewan** [comment]
The new name doesn't matter so much; how about just doitcss?

**Comment by csshugs** [comment]
“doitcss”? 🤔 Let me think about that...

😕

😏

🙂

😀

What I like about that is that:

1. It sounds kinda cool, because it comes out so fluidly out of my mouth
2. It references the ITCSS methodology directly in its name, but is not coupled too tight with it ([danger of confusion](https://github.com/inuitcss/inuitcss/issues/426#issuecomment-653760579))
3. The imperative form urges people to use it 😉 

I need to let that sink in further, but I think I like it!

**Comment by LayZeeDK** [comment]
How about `justdoitcss`? 😄

**Comment by glennmcewan** [comment]
> How about `justdoitcss`? 😄

I fear that might be another issue! 😆 

> I need to let that sink in further, but I think I like it!

Awesome! I wasn't so sure people would agree, but I think it makes sense with its simplicity and aptness to ITCSS.

**Comment by csshugs** [comment]
Okay, I slept over it 😄 I still think “doitcss” is by far the best idea for a new name!

What does @inuitcss/core think about it? And most importantly, @csswizardry ? After all, inuitcss is your brainchild, so I guess your opinion trumps (pun intended) them all.

**Comment by LayZeeDK** [comment]
Just throwing one last suggestion in:
```
getitcss
```
Get it?

**Comment by inlikealion** [comment]
@LayZeeDK See [#issuecomment-658392047](#issuecomment-658392047). Easily jumbled pronunciation produces inappropriate name with that one.

I'm personally in favor of a rename @csswizardry. I like the `doitcss` suggestion best so far, as it does loosely couple with the ideology of ITCSS. 

The first thought I had was `vcss`: its short and shows that inverted triangle idea without the complicated phraseology or possible nazi imagery. Possible drawback: might fight for attention with Vue and its various libraries, but I'm not TOO concerned about it.

**Comment by hayatbiralem** [comment]
I like both `doitcss` and `vcss`. Between these two my vote goes to `vcss` 👍

**Comment by LayZeeDK** [comment]
`itcsswizardry` 🧙

**Comment by dennisfrank** [comment]
I like doitcss as name too. 

Or some variant of it:

DoitCSS

DOITcss

DOIT CSS

Or just:

DOIT 

(And @csshugs writes the Doitorial 😙)

or:

DUIT

(Why not? 🙂)

`</brainstorm>`

**Comment by herzinger** [comment]
I like the `vcss` idea, but why did we never consider simply `itcss`?

**Comment by glennmcewan** [comment]
`itcss` is the paradigm / design system, so naming a framework around it with the same name could be confusing. `doitcss` is my suggestion so I'm biased of course, but I think it's a nice way to include `itcss` while keeping the name intuitive & relevant to its origins.

**Comment by fuhlig** [comment]
>... but why did we never consider simply `itcss`?

It was a suggestion in the initial post and the first part of this entire conversation is about ITCSS as a name for this framework

**Comment by herzinger** [comment]
Yes, I remember. This conversation is going on for months, I forgot about it... doitcss sounds inappropriate

**Comment by LayZeeDK** [comment]
> ITCSSFx: The ITCSS framework.

**Comment by cbleslie** [comment]
How long does it take to rename a project? More than a year apparently.

How about just abandoning the project, and archiving it?

**Comment by csshugs** [comment]
I’m sorry @cbleslie , but your comment is not any helpful of a contribution to this issue! To the first question you are asking, you are giving the answer yourself. And why should we archive the project? inuitcss is still used by many people and will continue to do so. It’s too bad, that it was once given an unfortunate name, but we cannot undo this fact for any recent versions of the framework. Currently, the project is in some kind of standby state (for personal reasons on my side). As far as I'm concerned, I have no doubt that the project will continue to be taken care of. And when that day comes, a rename will happen.

The fact that the project is currently placed on hold shouldn’t worry anyone. Technically, it is in a good shape to be used in professional projects — what doesn’t mean there aren’t things to be optimised.

If I’m dissatisfied with a product that I once loved, I favour giving feedback and providing my own help over unconstructive complaining that demotivates the responsible developers. I can fully understand, that people are upset or even frustrated about the current state of the project. But that doesn’t justify unfair cynicism or the like. I’d be happy and thankful if we could keep all this in a constructive, positive and respectful manner.

**Comment by cbleslie** [comment]
Wow, long response, with lots of low effort excuses, @csshugs. Make all the excuses you want. The longer the name stays the same, the longer you show that you're a insensitive dick.

You have personal problems, but personal problems are not an excuse for continued disparagement of a people. There are people out there that have personal problems, and they don't seem to let that get in their way of being a normal, decent, human. Don't make excuses. Don't be shitty.

> And why should we archive the project?

Rename it. Pick a name. Done. Live with the legacy, and show you can do better.

> It’s too bad, that it was once given an unfortunate name, but we cannot undo this fact for any recent versions of the framework. 

No one is asking you to undo the past. Move forward. Pick a name. Change it in the documentation, and be done with it.

> I’d be happy and thankful if we could keep all this in a constructive, positive and respectful manner.

You've have had plenty of time to be "constructive". Pick a fucking name. It takes less than 5 minutes to rename a repo. Almost any name that's been proposed in this thread is good enough. 

Jesus. The fucking privilege. Be better.

**Comment by aaronstezycki** [comment]
Such a childish and badly tempered response. No need for that. I would moderate that response, or be removed from contributing.

**Comment by glennmcewan** [comment]
Exchange Twitter handles & argue it out there...this will end up locked due to a flame war.

I agree with the sentiment to _just pick a name_ and how long this issue has been open for, but let's not be flippant about it. At this point we would really need @csswizardry's feedback on where to take this...

**Comment by DevJMD** [comment]
@cbleslie imagine being a fully-grown man, and writing what you wrote. Telling people to "be better" with that ironic response is the most laughable thing I've read today. Cheers for that!

> "personal problems are not an excuse for continued disparagement of a people"

Sounds like a you problem. Sorry, a _privileged_ problem. _Ironic_ world we live in, isn't it?

@csswizardry, please lock this, rename it, and remove the man-child before the toys fall out of the pram.

**Comment by aaronstezycki** [comment]
> This a really dirty and fast sketch, but we could use something along this lines, and let the branding visuals conciliate the question of the direction.
> ![ziggurat concept](https://user-images.githubusercontent.com/1102234/86641808-cbf26b00-bfb1-11ea-9f18-974d2891a0c1.png)

I think this has legs for a logo mark. Simple, easily recognisable, and could be given some different treatments for sister repos ... like plugins, themes, etc etc. Maybe dumb it down a bit, and thicken the borders for each ring, but I like this :)

**Comment by mtsweir** [comment]
The logo concept is interesting and could be a nice direction to take it, there are a few things I have comments on:
- The fine lines will be near impossible to see when scaled down e.g mobile, a good logo should be able to adapt and be versatile in various use cases.
- There are too many lines, I understand the concept but this could/should definitely be cut back to simply the shape, it feels busy visually which isn't the message I'd want to convey this as a system.
- Is there going to be a typeface as well? A symbol alone is just a small part of the brand.
- Is the colour just a concept colour?
- I can see this taken to a level where it's a 3D object with nice negative space around it and consisting of 3 layers that spiral downwards to convey a CSS cascade but keeping to a pyramid shape. It's just a thought though but could be something to consider here.

**Comment by herzinger** [comment]
As a said a bunch of times, this was not a brand suggestion, just a sketch of a concept. Branding takes a lot more time and effort, mostly to address the stuff you guys have outlined and a few other questions. The color used is just my sketching color. The line width is just the default brush size I had when I traced it. It's just a sketch, don't fixate on it. We never even decided on a name, I wouldn't put the effort into a professional brand at this point.

**Comment by krems04** [comment]
> Exchange Twitter handles & argue it out there...this will end up locked due to a flame war.
> 
> I agree with the sentiment to _just pick a name_ and how long this issue has been open for, but let's not be flippant about it. At this point we would really need @csswizardry's feedback on where to take this...

Did that flame war ever end up happening? Am I too late to this party? I just ended up encountering this project by chance, like the implementation and am also wondering how I can contribute. What happens from here?

**Comment by inlikealion** [comment]
Core: @anselmh, @csshugs, @csswizardry, @florianbouvot, @nenadjelovac

To recap, we won't call it `itcss` to avoid confusion with methodology. Rounding up the other names receiving most feedback:

* `doitcss`
* `vcss`

After rereading the entire thread, I'll toss out _ONE_ more option in the `vcss` vein: 
### valleycss 
Earlier there was traction on pyramid or mountain themes, but those are the wrong shape. Valley is the correct shape, is more specific than simply `vcss`, could provide more brand/theme inspiration because of that, plus the concept of valleys building up to mountains, etc.

CAN WE GET A VOTE by core members and move on? 

@csswizardry Care to make an executive decision? I'd love for us to get over the hump here so we can move on to finishing up `v7`. 

---

Addition: github.com/vcss is taken by a seemingly less active org with no public repos
I've setup org accounts for github.com/doitcss and github.com/valleycss, both of which also have open domains, to ensure they don't get sniped.

**Comment by dennisfrank** [comment]
I like valleycss. It makes sense.

**Comment by inlikealion** [comment]
Update: I see some core team giving thumbs-up (yay!), but not weighing in yet. 

I registered `valleycss` as an org on `npm`. Have not done the same for doitcss. I've also been gathering some cool valley inspo (California: LA, Coachella, etc; Sci-Fi retro, etc).

**Comment by herzinger** [comment]
I like valleycss, so I would go with that. I never liked doitcss because of the sexual connotation. Maybe with a different verb, like "use" or "make"... But either way, I like the valley idea much better.

**Comment by Uzume** [comment]
In homage to both ITCSS and @csswizardry, I suggest `itcsswiz` meaning it allows one to do ITCSS things easily. I terms of a logo I am thinking of an inverted triangular face with a wizard's hat.  The hat could perhaps be decorated with key bits of CSS written on the hat as a wizard's hat often has celestial symbols. The face would be an experienced and "wizened" one but of course also happy as everyone is happy to get their CSS work done more easily with the experienced tools of `itcsswiz`.
